src/components/data-table.jsx

```diff
@@ -155,9 +152,10 @@ const DataTable = ({
   getIdKey,
   selectable,
   onHeaderClick,
+  compact,
 }) => (
   <Fragment>
```

Comment:
Why is there a Fragment here, if this is rendering only one child at the root of this component?